package/MDAnalysis/core/groups.py

```diff
@@ -1809,9 +1831,20 @@ def __init__(self, ix, u):
         self._ix = ix
         self._u = u
 
+    def __setattr__(self, attr, value):
```

Comment:
Since this method and the equivalent one in `GroupBase` are almost identical, couldn't their code be factorized in an external function?